uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Reminder.php

```diff
@@ -32,9 +32,8 @@ private static function _setActivityOptions()
 
     private static function _checkRelatedExtensions()
     {
-        $appraisalExtension = CRM_Core_DAO::executeQuery("SELECT id FROM civicrm_extension WHERE full_name = 'uk.co.compucorp.civicrm.appraisals' AND is_active = 1 LIMIT 1");
-        $appraisalExtension->fetch();
-        if ($appraisalExtension->id)
+        $isEnabled = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Extension', 'uk.co.compucorp.civicrm.appraisals', 'is_active', 'full_name');
```

Comment:
I get some strange behaviors when using 'Extension' API entity. It lists extensions but seems like I can't get a single result for specified extension. Can you check on your local installation? Maybe I'm missing something.